source/4.0/guides/sql-how-to/order-by-nested-association-attribute.html.md

```diff
@@ -0,0 +1,49 @@
+---
+chapter: SQL How To
+title: Order with attribute of an associated record
+---
+This article assumes:
+
+* You have a `users` table
+* You have a `teams` table 
+
+We have Users, each users is associated to a Team. Let's imagine we want to get all the users, 
+but we want them ordered by Team name, and if they're in the same team, we want them to be 
+ordered by creation_date.
+
+``` ruby
+module Repositories
+  class Users
+    def get_ordered_by_team_and_creation_date
+      users
+        .combine(:team)
+        .order('team[:name]', :created_at)
```

Comment:
This does not look like a valid syntax. You would have to join teams to be able to order by one of the columns from team table.

Reply:
Oh yes, you're right, there's something messed up here 🤔 
I'm fixing this!